Purchases/FoundationExtensions/Error+Extensions.swift

```diff
@@ -27,9 +27,10 @@ extension Error {
         }
 
         let asNSError = self as NSError
+        let underlyingNSError = attachedError as NSError
```

Comment:
Maybe move this to the guard? `attachedError` isn't used:
```swift
guard let underlyingError = maybeError as NSError? else { }
```

Reply:
🤦‍♂️. Thank you